test/framework/integration_framework/test_irohad.hpp

```diff
@@ -77,6 +77,12 @@ namespace integration_framework {
         internal_server->shutdown();
       }
     }
+
+    void terminate(const std::chrono::system_clock::time_point &deadline) {
+      if (internal_server) {
+        internal_server->shutdown(deadline);
+      }
```

Comment:
Same for the else branch

Reply:
Same as above, nothing to terminate if there is no internal server.